app/services/api/map_hesa_attributes/placements/v01.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Api
+  module MapHesaAttributes
+    module Placements
+      class V01
+        ATTRIBUTES = %i[school_urn placement_days].freeze
+
+        def initialize(params)
+          @params = params
+        end
+
+        def call
+          mapped_params
+        end
+
+      private
+
+        def mapped_params
+          {
+            school_id:,
+            urn:,
+            name:,
+          }
+        end
+
+        def school_id
+          school_urn = @params[:school_urn]
```

Comment:
I _think_ that `school_urn` is right. It's referenced in the CreateFromHesa service [here](https://github.com/DFE-Digital/register-trainee-teachers/blob/9c177b2c936d79cad2eb4a41f07ad26eac4cebe7/app/services/placements/create_from_hesa.rb#L23) 

And the placement itself has a separate urn, so I think it would be confusing to make this urn. Let me know what you think.